tally/CMakeLists.txt

```diff
@@ -14,6 +14,8 @@ file(GLOB PUB_HEADERS ${PUB_HEADERS} "*.hpp")
 # Libraries
 set(LINK_LIBS)
 list(APPEND LINK_LIBS ${DAG_LINK_LIBS})
+list(APPEND LINK_LIBS ${GFORTRAN_LIBRARY})
+list(APPEND LINK_LIBS ${QUADMATH_LIBRARY})
```

Comment:
It was to do with how OS/X finds the libraries, but I had been working on a branch that didnt include your changes. Ill try without